packages/vsce/src/utils/profileManagement.ts

```diff
@@ -282,9 +268,8 @@ export class ProfileManagement {
   /**
    * Return all the regions in a given plex
    */
-  public static async getRegionInfoInPlex(plex: CICSPlexTree): Promise<any[]> {
+  public static async getRegionInfoInPlex(plex: CICSPlexTree, session: Session): Promise<any[]> {
```

Comment:
We can probably get the session (2nd arg) from the plex tree (1st arg) using `getParent` or `getSession`, or a combination, right? Although it's not exactly messy currently so optional.